cfgov/unprocessed/apps/owning-a-home/css/a-range.less

```diff
@@ -0,0 +1,115 @@
+/*
+    Range slider for credit score
+    --------------------------- */
+.a-range {
```

Comment:
not sure if this should be a molecule, instead of an atom

Reply:
I was going off of selects and inputs being atoms https://github.com/cfpb/capital-framework/tree/canary/src/cf-forms/src/atoms

Reply:
You may be right though with the slider label, since that brings it more in line with https://github.com/cfpb/capital-framework/blob/canary/src/cf-forms/src/molecules/form-fields.less

Reply:
Thinking about it further, I think it would be an atom. The label on the slider is part of the slider, as opposed to a form field which has an input and label that are separate entities. I'll merge this as an atom, but we can revisit no doubt if/when this moves to CF or into a more general component within cfgov-refresh.